cernopendata/modules/fixtures/data/docs/cms-getting-started-nanoaod/cms-getting-started-nanoaod.md

```diff
@@ -149,18 +149,18 @@ Before exiting the ROOT session, your histogram canvas should look something lik
 
 #### NanoAOD skimming and analysis with `NanoAOD-tools`
 
-The [NanoAOD-tools](https://github.com/cms-nanoAOD/nanoAOD-tools/) repository provides a toolkit to perform operations on NanoAOD files outside the CMSSW software environment. This program can be used to:
+The [NanoAOD-tools](/record/12507) repository provides a toolkit to perform operations on NanoAOD files outside the CMSSW software environment. This program can be used to:
 
```

Comment:
If we really go into details, the record is not a repository now so maybe we need to change to `The [NanoAOD-tools](https://github.com/cms-nanoAOD/nanoAOD-tools/) record`?

Reply:
Changing this one to "the nanoaod-tools software" (where the software can be found on that record)